src/core/manager.c

```diff
@@ -3112,11 +3112,9 @@ int manager_serialize(
                         continue;
 
                 t = manager_timestamp_to_string(q);
-                {
-                        char field[strlen(t) + STRLEN("-timestamp") + 1];
-                        strcpy(stpcpy(field, t), "-timestamp");
-                        dual_timestamp_serialize(f, field, m->timestamps + q);
-                }
+                char field[strlen(t) + STRLEN("-timestamp") + 1];
```

Comment:
I think this is the only place in the codebase where a VLA is used. I think `_cleanup_free_ char *field; field = strjoin(t, "-timestamp")` or the equivalent with `strjoina()` would be better.

Reply:
we use VLAs quite frequently actually. this is far from the only case.

Reply:
Sorry, I thought that the `STRLEN` macro was introduced to avoid variable length arrays.